dogowner/models.py

```diff
@@ -0,0 +1,78 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Gender(models.TextChoices):
+    Male = 'M', 'Male'
+    Female = 'F', 'Female'
+    Unknown = 'UN', 'Unknown'
+
+
+class Dogowner(models.Model):
+
+    user = models.ForeignKey(User,
```

Comment:
you should probably use `models.OneToOneField` instead.

Reply:
Fixed